### PR TITLE
fix(detail): Empty field can not be saved.

### PR DESCRIPTION
**Pull request by vikram-raj**
Issue: https://github.com/openshiftio/openshift.io/issues/2438

### CODE REVIEW

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2491-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2491-fabric8-planner.badger.fabric8.io

File: src/tests/specs/smoke/smokeTest.spec.ts [hunk 1]
```diff
@@ -50,6 +50,12 @@ describe('Planner Smoke Tests:', () => {
     expect(await planner.workItemList.hasWorkItem(c.updatedWorkItem.title)).toBeTruthy();
   });
 
+  it('Check WorkItem title is not empty', async () => {
```
Comment (sanbornsen) [inline src/tests/specs/smoke/smokeTest.spec.ts]: The test is about if you can save it when the title is empty. I'd re-work on the test message.

Comment (sanbornsen) [reply src/tests/specs/smoke/smokeTest.spec.ts]: Nice job adding a test.

Comment (vikram-raj) [reply src/tests/specs/smoke/smokeTest.spec.ts]: Done https://github.com/fabric8-ui/fabric8-planner/pull/2491/commits/0e0fd06766d06a02981ef5b3370b37a50b5ff6cf

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2491-10 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2491-fabric8-planner.badger.fabric8.io

**Comment by vikram-raj** [comment]
> There is an issue, Please follow these steps:
-Open quick preview and try saving empty title.
-We are getting error
-Now Close the quick Preview and again open Title is empty.

@Raunak1203 fix is issue in https://github.com/fabric8-ui/fabric8-planner/pull/2491/commits/12bbf6ef674b7d02007a445dca62cd127b9e87db

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2491-11 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2491-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@vikram-raj fabric8/fabric8-ui:SNAPSHOT-PR-2491-12 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2491-fabric8-planner.badger.fabric8.io

**Pull request merged**